sys/include/hashes/cmac.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2016 Fundación Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_hashes_cmac AES_CMAC
+ * @ingroup     sys_hashes
+ * @brief       Implementation of the AES_CMAC hashing function
+
+ * @{
+ *
+ * @file
+ * @brief       AES_CMAC interface definition
+ *
+ * @author      José Ignacio Alamos <jose.alamos@inria.cl>
+ */
+
+//TODO: Add source
+
+#include <stdio.h>
+#include "crypto/ciphers.h"
+
+#define CMAC_BLOCK_SIZE 16
+
+typedef struct {
+    /** AES128 context */
+    cipher_t aes_ctx;
+    /** auxiliar array for CMAC calculations **/
+    uint8_t X[CMAC_BLOCK_SIZE];
+    /** current last block **/
+    uint8_t M_last[CMAC_BLOCK_SIZE];
+    /** last byte in last block */
+    uint32_t M_n;
+} cmac_context_t;
+
+/**
+ * @brief Initialize CMAC message digest context
+ *
+ * @param[in] ctx     Pointer to the CMAC context to initialize
+ * @param[in] key     Key to be set
+ * @param[in] key     Size of the key
```

Comment:
should be keySize (generates a Doxygen warning). Also don't use LowerCamelCase but snake_case